starter_code/src/movies.js

```diff
@@ -1,20 +1,126 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
-
-
+function turnHoursToMinutes(arrayOfMovies){
+  var durationHours;
+  var durationMin;
+  var durArray=[];
+  var moviesInMinutes = arrayOfMovies.map((oneMovie)=> {
+  if (oneMovie.duration.indexOf("h") === -1 ){
+    durationHours = 0;
+    durArray = oneMovie.duration.split("min")
+    durationMin = Number(durArray[0]);
+    oneMovie.newDuration = durationMin;
+  }
+  else if (oneMovie.duration.indexOf("m") === -1){
+    durationMin = 0;
+    durArray = oneMovie.duration.split("h")
+    durationHours = Number(durArray[0]);
+    console.log(durationHours)
+    oneMovie.newDuration = durationHours * 60;
+  }
+  else{
+    durArray = oneMovie.duration.split("h ")
+    durArray[1]=durArray[1].slice(0, (0,durArray[1].length-3));
+    durationHours =Number(durArray[0])*60;
+    durationMin = Number(durArray[1]);
+    oneMovie.newDuration = (durationHours + durationMin);
+  }
+  let updatedMovie = {
+    title: oneMovie.title,
+    year: oneMovie.year,
+    director: oneMovie.director,
+    duration: oneMovie.newDuration,
+    genre: oneMovie.genre,
+    rate: oneMovie.rate
+  }
+  return updatedMovie;
+  })
+  return moviesInMinutes;
+}
 // Get the average of all rates with 2 decimals 
 
+function ratesAverage(arrayOfMovies) {
+  var sumOfRatings = arrayOfMovies.reduce((total, eachMovie)=>{
+    return total + Number(eachMovie.rate)
+  },0) 
+  return sumOfRatings/arrayOfMovies.length
+}
 
 // Get the average of Drama Movies
+function roundTwoDecimals(num) {
```

Comment:
There's method `.toFixed()` that receives one argument and if you pass number 2, it will round your number to 2 decimals.